forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/test/resources/bootstrap.properties

```diff
@@ -9,6 +9,3 @@ spring.data.mongodb.uri=mongodb://localhost:57019/test
 
 #disable JWS signature verification here - rely on the functional tests instead
 rs.detached-signature.enable=false
-
-# Enable the extended validations for VRPs for Integration Testing
-rs.vrp.extended.validation.enable=true
```

Comment:
Can we leave this documented in the properties file, even if it is set to false?

Reply:
Yes keep commented the line 14 to be documented please.
```yaml
# Enable the extended validations for VRPs for Integration Testing, default value 'true'
# - rs.vrp.extended.validation.enable=false 
```

Reply:
sure